config/routes.rb

```diff
@@ -2,6 +2,8 @@
   root 'site#index'
 
   get '/profile' => 'profile#show'
```

Comment:
If you use 3 routes for the same controller, why not to use `resources :profile' then?